src/harness.ts

```diff
@@ -167,18 +183,23 @@ class WidgetHarness<P extends WidgetProperties, W extends Constructor<WidgetBase
 	public lastRender: DNode | undefined;
 	public renderCount = 0;
 
-	constructor(widgetConstructor: W, afterCreate: (element: HTMLElement) => void) {
+	constructor(widgetConstructor: Constructor<W>, afterCreate: (element: HTMLElement) => void, metaData: WeakMap<Constructor<WidgetMetaBase>, MetaData>) {
 		super();
 
-		this._widgetConstructor = SpyRenderMixin(widgetConstructor, this);
+		this._widgetConstructor = SpyWidgetMixin(widgetConstructor, this);
 		this._afterCreate = afterCreate;
+		this._metaData = metaData;
+	}
+
+	protected onElementCreated(element: HTMLElement, key: string) {
+		this._afterCreate(element);
 	}
 
 	/**
 	 * Called by a harnessed widget's render spy, allowing potential assertion of the render
 	 * @param actual The render, just after `afterRender`
 	 */
-	actualRender(actual: DNode) {
+	public actualRender(actual: DNode) {
```

Comment:
Why does this have to be explicitly marked as `public`?

Reply:
It is unnecessary, though we have been trying to be more explicit in the visibility operators.  I can remove it if it is a concern.  I forgot I did it.